src/apple/app_store/process.rs

```diff
@@ -79,4 +79,8 @@ impl ProcessExt for Process {
     fn group_id(&self) -> Option<Gid> {
         None
     }
+
+    fn wait(&self) {
+        None
```

Comment:
Should not return anything.

Reply:
Should I keep it like this?
```rust
fn wait(&self) {

}
```

Reply:
Yes and run `cargo fmt` afterwards, you'll see. ;)

Reply:
haha thanks